shanoir-ng-datasets/src/main/java/org/shanoir/ng/dataset/service/DatasetAsyncService.java

```diff
@@ -0,0 +1,17 @@
+package org.shanoir.ng.dataset.service;
+
+import org.shanoir.ng.dataset.model.Dataset;
+import org.shanoir.ng.datasetfile.DatasetFile;
+import org.shanoir.ng.shared.exception.ShanoirException;
+import org.springframework.security.access.prepost.PreAuthorize;
+
+import java.util.List;
+
+public interface DatasetAsyncService {
+
+    @PreAuthorize("hasRole('ADMIN') or (hasRole('EXPERT') and @datasetSecurityService.hasRightOnDataset(#datasetId, 'CAN_ADMINISTRATE'))")
+    void deleteDatasetFilesFromDiskAndPacs(List<DatasetFile> datasetFiles, boolean isDicom, Long datasetId) throws ShanoirException;
```

Comment:
Hi @pierrehenri-dauvergne, thank you for this great PR.
I have the impression this method is not necessary?
With kind regards, Michael